src/v/storage/failure_probes.h

```diff
@@ -32,8 +32,8 @@ class log_failure_probes final : public finjector::probe {
     enum class methods : type {
         append = 1,
         roll = 2,
-        truncate = 3,
-        truncate_prefix = 4
+        truncate = 4,
+        truncate_prefix = 8
```

Comment:
nice catch. we should have had some test that verified something simple like that if `truncate` is set then no other methods are set.

Reply:
Yeah I'd like to add a unit test here.. trying not to get too distracted for now.